hexa/databases/api.py

```diff
@@ -0,0 +1,71 @@
+import psycopg2
+from django.conf import settings
+from psycopg2 import sql
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+
+def get_db_server_credentials():
+    return {
+        "role": settings.WORKSPACE_DATABASE_ROLE,
+        "password": settings.WORKSPACE_DATABASE_PASSWORD,
+        "host": settings.WORKSPACE_DATABASE_HOST,
+        "port": settings.WORKSPACE_DATABASE_PORT,
+    }
+
+
+def check_db_name(name):
+    if not name:
+        raise ValueError("rmpty value for database name")
```

Comment:
Can you add typing info to your functions? Not only this one, in general

Reply:
> Almost there +1 - see comments.
> 
> Also, I don't see the CREATE EXTENSION statements, am I missing something?

Oh not yet. I'll add it